.github/workflows/_rust.yml

```diff
@@ -126,11 +126,17 @@ jobs:
       fail-fast: false
       matrix:
         runs-on:
-          # TODO
-          # - ubuntu-20.04
+          - ubuntu-20.04
           - windows-2019
+        include:
+          - runs-on: ubuntu-20.04
+            os-name: linux
```

Comment:
I tried to get rid of this but it's easier to control the artifact names this way than with `runner.os` which has caps in it, e.g. "Linux" instead of "linux"